spec.md

```diff
@@ -1181,10 +1265,11 @@ Errors](#service-broker-errors).
 
 | Response Field | Type | Description |
 | --- | --- | --- |
-| credentials | object | A free-form hash of credentials that can be used by applications or users to access the service. |
-| syslog_drain_url | string | A URL to which logs MUST be streamed. `"requires":["syslog_drain"]` MUST be declared in the [Catalog](#catalog-management) endpoint or the platform MUST consider the response invalid. |
+| credentials | object | A free-form hash of credentials that can be used by applications or users to access the service. MUST be returned for a synchronous request if the Service Broker supports generation of credentials. |
```

Comment:
let's drop "synch" from all of these since its misleading - the properties apply to both sync and async

Reply:
Do they? I thought for async, the GET endpoint should be used to fetch credentials?

Reply:
Yes, but I'd prefer if we didn't have to qualify so many sentences with "sync" vs "async" since it then leaves the reader wonder what should happen in the opposite case. Which means we get very verbose or repetitive. I think it would be cleaner to just say "this json is what is returned from this op. Whether you get it on the initial response (sync) or later during a GET (async) doesn't change the format of the json".